### PR TITLE
Test with Rails 7

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Review by camillevilla — APPROVED** [review]
✌️ Thanks for cleaning out the cruft

File: test/dummy/config/storage.yml [hunk 1]
```diff
@@ -1,34 +1,3 @@
...
-#   service: AzureStorage
-#   storage_account_name: your_account_name
-#   storage_access_key: <%= Rails.application.credentials.dig(:azure_storage, :storage_access_key) %>
-#   container: your_container_name
-
-# mirror:
-#   service: Mirror
-#   primary: local
-#   mirrors: [ amazon, google, microsoft ]
+  root: /Users/cabeer/Projects/sul-dlss/access/global_alerts/tmp/storage
```
Comment (thatbudakguy) [inline test/dummy/config/storage.yml:3]: does it matter that this machine-local path is here?

**Review by thatbudakguy — APPROVED** [review]
lots of deleted lines/files makes me happy 🙂

File: test/dummy/config/storage.yml (hunk 1 above)
Comment (cbeer) [reply test/dummy/config/storage.yml:3]: Probably not... we're not using activestorage anyway, and... there's no real tests for this repo either 😬

**Pull request merged**